src/njoy21/input/PLOTR/Card6/Ystep.hpp

```diff
@@ -7,16 +7,16 @@ struct Ystep {
       "The default behavior is automatic scaling.  The value is ignored if\n"
       "log scaling is used.";
   }
-  static Value_t defaultValue( const Value_t yl ){
-    if( yl != std::nullopt ){
-      Log::info( "When using default values in PLOTR::Card6, the default\n"
-                 "values should be used for all parameters.");
-      std::exception e;
-      throw e;
-    }
+  static Value_t defaultValue(){ // const Value_t yl ){
+//    if( yl != std::nullopt ){
```

Comment:
I think this is another case where the test input is wrong and needs to be changed. It says that all three values must be there—or none.